test/datapackage.js

```diff
@@ -185,13 +185,23 @@ describe('Datapackage', () => {
     })
   })
 
-  describe('remote datapackage', () => {
-    it('reads the resource', async () => {
+  describe('remote datapackage resources', () => {
+    it('relative resource', async () => {
       const descriptor = 'https://raw.githubusercontent.com/frictionlessdata/datapackage-js/master/data/dp1/datapackage.json'
 
       const datapackage = await new Datapackage(descriptor)
-      const table2 = await datapackage.resources[0].table
-      const data = await table2.read()
+      const table = await datapackage.resources[0].table
+      const data = await table.read()
+
+      assert(_.isEqual(data, [['gb', 100], ['us', 200], ['cn', 300]]), 'Invalid data.')
+    })
+
+    it.only('URL resource', async () => {
+      const descriptor = 'https://dev.keitaro.info/dpkjs/datapackage.json'
```

Comment:
The URL is temporary, once this is merged this will be changed to the newly added `data/dp4-remote-resource/datapackage.json`